typescript/src/index.ts

```diff
@@ -20,11 +25,19 @@ const TBTC = {
   getRevealedDeposit,
   requestRedemption,
   getRedemptionRequest,
+  requestOptimisticMint,
+  getOptimisticMintingRequest,
```

Comment:
I'm wondering about exposing all optimistic-minting-related functions in a separate namespace. Initially, the plan was:
- `TBTC` namespace exposes all functions relevant for regular TBTC system clients interested in deposits and redemptions (e.g. tBTC dApp)
- `SpvMaintainer` targets the maintainers of the system that will be responsible for submitting SPV proofs of Bitcoin transactions performed by the wallets

The `requestOptimisticMint` and `getOptimisticMintingRequest` will be used by Minters and Guardians so they don't quite match the `TBTC` namespace. I think we have options like:

 **Option 1**
```
const OptimisticMinting = {
    requestOptimisticMint
    getOptimisticMintingRequest
    // other functions specific to both Minters and Guardians
}
```

**Option 2**
```
const Minter = {
    requestOptimisticMint
    getOptimisticMintingRequest
    // other functions specific to Minters
}

const Guardian = {
    // functions specific to Guardians
}
```
What do you think?

Reply:
It makes sense. I prefer option #1.